tasks/eslintCI.js

```diff
@@ -0,0 +1,47 @@
+// @noflow
+/* eslint-disable no-console */
+const { ESLint } = require("eslint");
+const { Git } = require("@adeira/monorepo-utils");
+const path = require("path");
+const log = require("fancy-log");
+
+async function eslintCI() {
+  const eslint = new ESLint({
+    reportUnusedDisableDirectives: "error",
+  });
+
+  function shouldLintAll(fileName) {
+    return [
+      path.basename(__filename),
+      "package.json",
+      "yarn.lock",
+      ".eslintrc",
+      ".eslintrc.js",
+      ".eslintignore",
+      "eslintJS.js",
+      "eslintTS.js",
+    ].some(f => fileName.endsWith(f));
+  }
+
+  const changedFiles = Git.getChangedFiles();
+
+  let filesToLint;
+
+  if (changedFiles.some(shouldLintAll)) {
+    log("Some critical files have been changed, running ESLint on all files...");
+    filesToLint = ".";
+  } else {
+    filesToLint = changedFiles.filter(fileName => fileName.match(/\.(jsx?|tsx?|flow)$/));
+  }
+
+  const results = await eslint.lintFiles(filesToLint);
+  const errorResults = ESLint.getErrorResults(results); // --quiet
+
+  if (results.errorCount > 0) {
+    const formatter = await eslint.loadFormatter();
+    console.log(formatter(errorResults));
```

Comment:
maybe here we can use `log.error(...)` from `fancy-log` as we already imported that module :) Or it does not make sense to use with `eslint.loadFormatter()`? 🤔 

Everything works good :)

Reply:
Thanks for the suggestion, but I think it's best to use `fancy-log` for single line outputs, this will span multiple lines.